### PR TITLE
feat: update charts to be able to leverage external static IP addresses

### LINKED ISSUE #636: ChartTLS: update charts to be able to leverage external static IP addresses

### PR DESCRIPTION
**Pull request by jeromy-cannon**
## Description

This pull request changes the following:

- update charts to be able to leverage external static IP addresses
- update fsnetman e2e test to set chart directory
- update envoy gateway to v0.6.0
- update cert-manager to v1.13.3
- update GatewayClass, HTTPRoute, and Gateway CRD to gateway.networking.k8s.io/v1
- move acme-cluster-issuer and self-signed from `fullstack-cluster-setup` to `fullstack-deployment`
- default selfSignedClusterIssuer to false in `fullstack-cluster-setup`
- updated acme-cluster-issuer to handle http01 with either ingres or gatewayHTTPRoute
- added loadBalancerIP and loadBalancerEnabled values for `fullstack-deployment`
- default grpcs to false for now, there is a separate issue to get this working: #632
- updated fsnetman helm install command to output syserr when in verbose mode
- updated GitHub workflow job NodeJS CLI E2E Tests to also run TLS install scenario (currently no E2E test, only install with options) 

### Related Issues

- Closes #636

### CODE REVIEW

**Review by leninmehedy — APPROVED** [review]
- Need to handle codacy errors later in a separate PR
- Need to sort out the Snyk Scan error in a separate PR

File: .github/workflows/zxc-fsnetman-tests.yaml [hunk 1]
```diff
@@ -130,9 +130,20 @@ jobs:
...
+          fsnetman init -d ../charts
+          fsnetman cluster create
+          fsnetman cluster setup -d ../charts \
+            --cert-manager \
+            --cert-manager-crds
+          fsnetman chart install -d ../charts \
+            --enable-tls \
+            --self-signed \
+            --enable-hedera-explorer-tls
+
```
Comment (leninmehedy) [inline .github/workflows/zxc-fsnetman-tests.yaml:146]: After setting up the chart again, don't we need to do some e2e testing?

Comment (leninmehedy) [reply .github/workflows/zxc-fsnetman-tests.yaml:146]: Seems you can just remove lines 131 ~135 and then just add  `npm run test-e2e`. That way you setup the network as required before running integration tests

Comment (jeromy-cannon) [reply .github/workflows/zxc-fsnetman-tests.yaml]: done

**Pull request merged**